Makefile

```diff
@@ -18,9 +18,11 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.12
 DOCKER_IMAGE_DEVELOPER_TOOLS := cft/developer-tools
 REGISTRY_URL := gcr.io/cloud-foundation-cicd
+ENABLE_BPMETADATA := 1
```

Comment:
We will need to add this via lint_env to enable it in CI
https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/f8107c787755d9ae0ef3b0e811e8b4bfd6009a7a/infra/terraform/test-org/org/locals.tf#L68

This is currently being skipped https://github.com/GoogleCloudPlatform/terraform-google-three-tier-web-app/actions/runs/4996523829/jobs/8949814716?pr=44

Reply:
If this is the standard practice, we could consider adding support for scrapping this value similar to `DOCKER_IMAGE_DEVELOPER_TOOLS` and `REGISTRY_URL`

Reply:
We may eventually do that after we let it soak with JSS repos. Right now I'll just add it to `lint_env`